packages/api/post-street-manager.ts

```diff
@@ -0,0 +1,78 @@
+import { APIGatewayEvent } from "aws-lambda";
+import { S3Client, PutObjectCommand } from "@aws-sdk/client-s3";
+import * as logger from "lambda-log";
+import { snsMessageSchema } from "./utils/snsMesssageTypes.zod";
+import { confirmSubscription, isValidSignature } from "./utils/snsMessageValidator";
+
+const allowedTopicArns = [
+    "arn:aws:sns:eu-west-2:287813576808:prod-activity-topic",
+    "arn:aws:sns:eu-west-2:287813576808:prod-permit-topic",
+    "arn:aws:sns:eu-west-2:287813576808:prod-section-58-topic",
+    "arn:aws:sns:eu-west-2:899289342948:test-street-manager-topic",
```

Comment:
We might want to take this one out before merging

Reply:
woops, forgot to remove that after testing my changes! removed